MAUI/Radio-Button/Visual-Customization.md

```diff
@@ -179,4 +179,87 @@ The [`ControlSize`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.
 	this.Content = stackLayout;
 
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+## EnabledAnimation
+
+The `EnabledAnimation` property is used to enable or disable the animation for the [`RadioButton`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Buttons.SfRadioButton.html) control. 
+
+{% tabs %}
+{% highlight xaml %}
+
+ <StackLayout>
+     <syncfusion:SfRadioButton Text="RadioButton EnabledAnimation True" EnabledAnimation="True"/>
+     <syncfusion:SfRadioButton Text="RadioButton EnabledAnimation False" EnabledAnimation="False"/>
+ </StackLayout>
+
+{% endhighlight %}
+{% highlight c# %}
+
+	StackLayout stackLayout = new StackLayout();
+	SfRadioButton radioButtonEnabled = new SfRadioButton
+	{
+		Text = "RadioButton EnabledAnimation True",
+		EnabledAnimation = true
+	};
+
+	SfRadioButton radioButtonDisabled = new SfRadioButton
+	{
+		Text = "RadioButton EnabledAnimation False",
+		EnabledAnimation = false
+	};
+
+	stackLayout.Children.Add(radioButtonEnabled);
+	stackLayout.Children.Add(radioButtonDisabled);
+
+	this.Content = stackLayout;
+
+{% endhighlight %}
+{% endtabs %}
+
+![.NET MAUI Radio Button EnabledAnimation](Images/VisualCustomization/enabledanimation.png)
+
+## SelectedValue
```

Comment:
move this section to the grouping.

Reply:
Moved the Selected value section to the grouping